webpack/assets/javascripts/react_app/components/HostsIndex/BulkActions/changeOwner/BulkChangeOwnerModal.js

```diff
@@ -0,0 +1,258 @@
+      isLoading={
+        userStatus === STATUS.PENDING || usergroupStatus === STATUS.PENDING
+      }
+    >
+      {__('Change Owner')}
+    </Button>,
+    <Button
+      key="cancel"
+      ouiaId="bulk-change-owner-modal-cancel-button"
+      variant="link"
+      onClick={handleModalClose}
+    >
+      {__('Cancel')}
+    </Button>,
+  ];
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      onClose={handleModalClose}
+      onEscapePress={handleModalClose}
+      title={__('Change Owner')}
+      width="50%"
+      position="top"
+      actions={modalActions}
+      id="bulk-change-owner-modal"
+      key="bulk-change-owner-modal"
+      ouiaId="bulk-change-owner-modal"
+    >
+      <TextContent>
+        <Text ouiaId="bulk-change-owner-options">
+          {__(
+            'Changing the owner will affect all of your selected hosts. Some hosts may already have been associated with the selected owner.'
+          )}
+        </Text>
+      </TextContent>
+      {userStatus === STATUS.RESOLVED && usergroupStatus === STATUS.RESOLVED && (
+        <Select
+          id="single-grouped-select"
+          isOpen={ownerSelectOpen}
+          selected={ownerId}
+          onSelect={handleOwnerSelect}
+          onOpenChange={isSelectOpen => setOwnerSelectOpen(isSelectOpen)}
+          toggle={toggle}
+          shouldFocusToggleOnSelect
+          ouiaId="bulk-change-owner-select"
+        >
+          {users && (
+            <SelectGroup label="Users">
+              <SelectList>
+                {users.results?.map(u => (
+                  <SelectOption key={`${u.id}-Users`} value={`${u.id}-Users`}>
+                    {u.login}
+                  </SelectOption>
+                ))}
+              </SelectList>
+            </SelectGroup>
+          )}
+          {usergroups && (
+            // eslint-disable-next-line spellcheck/spell-checker
+            <SelectGroup label="Usergroups">
+              <SelectList>
+                {usergroups?.results?.map(ug => (
+                  <SelectOption
+                    // eslint-disable-next-line spellcheck/spell-checker
+                    key={`${ug.id}-Usergroups`}
+                    // eslint-disable-next-line spellcheck/spell-checker
+                    value={`${ug.id}-Usergroups`}
+                  >
+                    {ug.name}
+                  </SelectOption>
+                ))}
+              </SelectList>
+            </SelectGroup>
+          )}
+        </Select>
+      )}
+      <div className="bulk-change-owner-modal__tree-view-container">
+        <FormattedMessage
+          id="selected-hosts-message"
+          defaultMessage="{count, plural, one {Selected host: {boldCount}} other {Selected hosts: {boldCount}}}"
+          values={{
+            count: selectAllHostsMode ? __('All') : selectedCount,
+            boldCount: (
+              <strong>{selectAllHostsMode ? __('All') : selectedCount}</strong>
+            ),
+          }}
+        />
```

Comment:
What I actually meant was,

1. eliminate this text here
2. Change the text above the modal to read, e.g. "'Changing the owner will affect all 2 of your selected hosts. Some hosts may already have been associated with the selected owner.'"  <---- and the "2" is bold. That's what the FormattedMessage is for.

Reply:
Fixed up